pkg/util/indexparams/disk_index_params_test.go

```diff
@@ -125,18 +125,25 @@ func TestDiskIndexParams(t *testing.T) {
 	})
 
 	t.Run("set disk index build params", func(t *testing.T) {
+		var params paramtable.ComponentParam
+		params.Init(paramtable.NewBaseTable(paramtable.SkipRemote(true)))
+
 		indexParams := make(map[string]string)
 		indexParams[PQCodeBudgetRatioKey] = "0.125"
 		indexParams[NumBuildThreadRatioKey] = "1.0"
 
 		indexParams[SearchCacheBudgetRatioKey] = "0.125"
-		err := SetDiskIndexBuildParams(indexParams, 100)
+		err := SetDiskIndexBuildParams(&params, indexParams, 100)
 		assert.NoError(t, err)
 
 		indexParams[SearchCacheBudgetRatioKey] = "aabb"
-		err = SetDiskIndexBuildParams(indexParams, 100)
+		err = SetDiskIndexBuildParams(&params, indexParams, 100)
 		assert.Error(t, err)
 
+		delete(indexParams, SearchCacheBudgetRatioKey)
+		err = SetDiskIndexBuildParams(&params, indexParams, 100)
+		assert.NoError(t, err)
```

Comment:
Please check if the value of SearchCacheBudgetRatioKey is equal to the default.

Reply:
`SearchCacheBudgetRatioKey` can not be checked in the test because we have no way to get it here.

Reply:
You will assign the default value to the `indexParams`, so just check if it worked.

Reply:
okay